packages/design-system-scripts/gulp/build.js

```diff
@@ -82,37 +80,6 @@ async function copyAll(dir, options) {
   return Promise.all(copyTasks);
 }
 
-/**
- * Used to generate typescript definition files for the core
- */
-/* eslint-disable */
-async function generateTypeDefinitionsFromPropTypes(dir) {
-  const src = path.join(dir, 'src', 'components');
-  const srcGlob = getSrcGlob(src);
-
-  return streamPromise(
-    gulp
-      .src(srcGlob, { base: src })
-      .pipe(
-        through.obj((file, enc, cb) => {
-          // Replace React component files with definitions, avoid modifying entry point
-          if (file.basename !== 'index.js' || file.dirname.split('/').pop() !== 'components') {
-            const definition = react2dts.generateFromFile(null, file.path);
-            file.contents = Buffer.from(definition);
-          }
-
-          file.extname = '.d.ts';
-          cb(null, file);
-        })
-      )
-      .pipe(gulp.dest(path.join(dir, 'dist', 'types')))
-      .on('finish', function () {
-        logTask('📜 ', 'Core Typescript definition files generated');
-      })
-  );
-}
-/* eslint-enable */
-
```

Comment:
Can you talk more as to why this function is no longer needed?

Reply:
Originally it was used to generate the Typescript definition files automatically, but now we are manually updating them, so its not being used anywhere anyway